src/bootstrap/config.rs

```diff
@@ -658,6 +660,7 @@ impl Config {
         config.npm = build.npm.map(PathBuf::from);
         config.gdb = build.gdb.map(PathBuf::from);
         config.python = build.python.map(PathBuf::from);
+        config.cmake = build.cmake.map(PathBuf::from);
```

Comment:
I think it probably makes sense for us to check that the CMAKE env variable is either (a) not set or (b) equivalent to the config.toml. If it is set and different, I think we should probably prefer config.toml still but issue a warning (just eprintln! is fine).

Can you add that here?